spec/active_interaction/integration/array_interaction_spec.rb

```diff
@@ -57,6 +57,24 @@ class CollectionProxy
     end
   end
 
+  context 'with a nested hash' do
+    let(:interaction) do
+      Class.new(ActiveInteraction::Base) do
+        array :foo do
+          hash do
+            integer :bar
+          end
+        end
+      end
+    end
+
+    let(:outcome) { interaction.run(foo: [{ bar: 'baz' }]) }
+
+    it 'sets error with nesting index' do
+      expect(outcome.errors).to include(:'foo.0.bar')
```

Comment:
It looks like Rails is doing this with brackets like `foo[0].bar`. https://blog.bigbinary.com/2016/07/07/errors-can-be-indexed-with-nested-attrbutes-in-rails-5.html

I try to stay in line with the Rails way as much as possible.

Reply:
so, @AaronLasseigne , as far as I understand it should be changed from current form to `array_attribute[index].hash_key` form, am I right?

Reply:
error key format was changed according to the latest comment